electrumsv/nodeapi.py

```diff
@@ -333,6 +339,14 @@ async def execute_jsonrpc_call_async(request: web.Request, object_data: Any) \
                 error=ErrorDict(code=INVALID_REQUEST,
                     message="Params must be an array or object"))))
 
+    wallet = get_wallet_from_request(request, request_id)
```

Comment:
Use `app_state.daemon.network` not `wallet`, and move this up to the top of the function if possible.

Reply:
Force pushed